recipe/meta.yaml

```diff
@@ -12,15 +12,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Well, maybe one needs to define `PYTHON` somewhere....

Reply:
I followed the recipe at https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip , thinking that ``PYTHON`` would be automagically defined.